packages/jest-cli/src/reporters/BaseReporter.js

```diff
@@ -28,20 +28,20 @@ class BaseReporter {
     results: AggregatedResult,
   ) {}
 
+  onTestStart(
+    config: Config,
+    path: Path,
+    runnerContext: RunnerContext,
+  ) {}
+
   onRunComplete(
     config: Config,
     aggregatedResults: AggregatedResult,
     runnerContext: RunnerContext,
   ): ?Promise<any> {}
 
   _write(string: string) {
-    // If we write more than one character at a time it is possible that
-    // node exits in the middle of printing the result.
-    // If you are reading this and you are from the future, this might not
-    // be true any more.
-    for (let i = 0; i < string.length; i++) {
-      process.stderr.write(string.charAt(i));
-    }
+    process.stderr.write(string);
```

Comment:
@dmitriiabramov can you explain this? We had a huge issue in the past with previous node versions where on www the test run would just exit if there was a large amount of failures. You could potentially try this by failing 10 % of all assertions and running all tests on www. If the test run passes and prints big diffs, it is safe to remove it.